tests/metrics/lib/common.bash

```diff
@@ -505,6 +508,28 @@ function get_current_kata_config_file() {
 	current_config_file="${KATA_CONFIG_FNAME}"
 }
 
+# This function checks if the current session is runnin as root, 
+# if that is not the case, the function exits with an error message.
 function check_if_root() {
 	[ "$EUID" -ne 0 ] && die "Please run as root or use sudo."
 }
+
+# This function launches a kata container using a Busybox image,
+# then collects the current number of vcpus and the free memory from the container.
+# Finalliy fullfills the global variables 'MEASURED_CONTAINER_NUM_VCPUS' and 'MEASURED_CONTAINER_FREE_MEM'
+function get_kata_memory_and_vcpus() {
+	local busybox_img="docker.io/library/busybox:latest"
```

Comment:
Is quay.io less restrictive than docker.io when it comes to pulling images without logging in?

Reply:
@danmihai1 thank you, you are correct, so I update the PR with your suggestion.